### PR TITLE
add Ukrainian lang & fix Russian typo

### PR DESCRIPTION
**Pull request by vshepel**

### CODE REVIEW

File: lang/uk/lang.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<?php
+
+return [
```
Comment (LukeTowers) [inline lang/uk/lang.php:3]:
```suggestion
<?php return [
```

File: lang/uk/lang.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        'code' => 'Код',
+        'code_comment' => 'Введіть код пункту меню, якщо хочете мати до нього доступ через API.'
+    ],
+    'content' => [
+        'menu_label' => 'Зміст',
+        'cant_save_to_dir' => 'Збереження файлів змісту в директорію static-pages заборонено.'
+    ],
+    'sidebar' => [
+        'Add' => 'Додати',
+        'search' => 'Пошук...'
```
Comment (LukeTowers) [inline lang/uk/lang.php:70]: Please make sure all elements have trailing commas

Comment (vshepel) [reply lang/uk/lang.php]: I copied the original ru/lang file and they weren't there. But, ok. I corrected both

**Comment by LukeTowers** [comment]
LGTM!

**Pull request merged**